pkgs/applications/networking/browsers/ungoogled-chromium/default.nix

```diff
@@ -162,13 +151,6 @@ let
       ''
     else browser;
 
-  optionalVaapiFlags = if useVaapi # TODO: Remove after 20.09:
```

Comment:
This removal part of a commit that was cherry-picked, but I think it's fine considering ungoogled-chromium was never a part of NixOS 20.03, unlike chromium, so this error doesn't really serve a purpose.